src/Traversal/Sdk/Sdk.targets

```diff
@@ -127,6 +127,7 @@
     <MSBuild Projects="@(ProjectReference)"
              Condition="'%(ProjectReference.Build)' != 'false'"
              BuildInParallel="$([MSBuild]::ValueOrDefault('%(ProjectReference.BuildInParallel)', '$(BuildInParallel)'))"
+             Properties="%(ProjectReference.SetConfiguration); %(ProjectReference.SetPlatform); %(ProjectReference.SetTargetFramework)"
```

Comment:
Change looks good, do you mind adding a unit test for each property to make sure it doesn't regress?

Reply:
Oh, yes!

Reply:
I've added tests now. These fail before my change.

Reply:
I'm looking into it...